src/ts/sqlite-functions.ts

```diff
@@ -40,95 +40,95 @@ namespace Module {
+        : (pStatement: ptr<sqlite3_stmt>, index: number) => number
         = Module["cwrap"]("sqlite3_column_double", "number", ["number", "number"])
 
     export const sqlite3_column_text
-        : (pStatement: ptr<sqlite3>, index: number) => string
+        : (pStatement: ptr<sqlite3_stmt>, index: number) => string
         = Module["cwrap"]("sqlite3_column_text", "string", ["number", "number"])
 
     export const sqlite3_column_blob
-        : (pStatement: ptr<sqlite3>, index: number) => number
+        : (pStatement: ptr<sqlite3_stmt>, index: number) => number
         = Module["cwrap"]("sqlite3_column_blob", "number", ["number", "number"])
 
     export const sqlite3_bind_text
-        : (pStatement: ptr<sqlite3>, index: number, val: string, n: number, pFree: ptr<sqlite3>) => SQLiteResult
+        : (pStatement: ptr<sqlite3_stmt>, index: number, val: string, n: number, pFree: ptr<sqlite3>) => SQLiteResult
         = Module["cwrap"]("sqlite3_bind_text", "number", ["number", "number", "string", "number", "number"])
 
     export const sqlite3_bind_int
-        : (pStatement: ptr<sqlite3>, index: number, val: number) => SQLiteResult
+        : (pStatement: ptr<sqlite3_stmt>, index: number, val: number) => SQLiteResult
         = Module["cwrap"]("sqlite3_bind_int", "number", ["number", "number", "number"])
 
     export const sqlite3_bind_int64ptr
-        : (pStatement: ptr<sqlite3>, index: number, val: Uint8Array) => SQLiteResult
+        : (pStatement: ptr<sqlite3_stmt>, index: number, val: Uint8Array) => SQLiteResult
         = Module["cwrap"]("sqlite3_bind_int64ptr", "number", ["number", "number", "array"])
 
     export const sqlite3_bind_double
-        : (pStatement: ptr<sqlite3>, index: number, val: number) => SQLiteResult
+        : (pStatement: ptr<sqlite3_stmt>, index: number, val: number) => SQLiteResult
         = Module["cwrap"]("sqlite3_bind_double", "number", ["number", "number", "number"])
 
     export const sqlite3_bind_null
-        : (pStatement: ptr<sqlite3>, index: number) => SQLiteResult
+        : (pStatement: ptr<sqlite3_stmt>, index: number) => SQLiteResult
         = Module["cwrap"]("sqlite3_bind_null", "number", ["number", "number"])
 
     export const sqlite3_bind_parameter_index
-        : (pStatement: ptr<sqlite3>, name: string) => number
+        : (pStatement: ptr<sqlite3_stmt>, name: string) => number
         = Module["cwrap"]("sqlite3_bind_parameter_index", "number", ["number", "string"])
 
     export const sqlite3_bind_blob
-        : (pStatement: ptr<sqlite3>, index: number, value: number, length: number, pCallback: number) => SQLiteResult
+        : (pStatement: ptr<sqlite3_stmt>, index: number, value: number, length: number, pCallback: number) => SQLiteResult
         = Module["cwrap"]("sqlite3_bind_blob", "number", ["number", "number", "number", "number", "number"])
 
     export const sqlite3_changes
-        : (pStatement: ptr<sqlite3>) => number
+        : (pDb: ptr<sqlite3>) => number
```

Comment:
I'm not sure this is the right change, considering this line: https://github.com/mackyle/sqlite/blob/997e2f8cf24acfa2d0e37523713ca0c2b021ce8e/src/main.c#L976. Were you targeting another method ?

Reply:
My bad! I misread. Thanks!